src/husarnet_transport.cpp

```diff
@@ -0,0 +1,132 @@
+#if defined(ESP32) && defined(HUSARNET)
+#include <Arduino.h>
+#include <Husarnet.h>
+#include <HusarnetClient.h>
+#include <micro_ros_arduino.h>
+
+extern "C" {
+
+static HusarnetClient client;
+
+bool arduino_husarnet_transport_open(struct uxrCustomTransport *transport) {
+  struct micro_ros_agent_locator *locator =
+      (struct micro_ros_agent_locator *)transport->args;
+
+  Serial1.printf("Connecting to \"%s:%d\"... ", locator->hostname,
+                 locator->port);
```

Comment:
```suggestion
```

IMO is better to not assume that the user wants stdout in Serial.

Reply:
agree, can be removed